tests.py

```diff
@@ -46,20 +47,31 @@ def test_flake8(self):
         assert exit_status == 0
 
     def test_additional_management_command_options(self):
-        exit_status, output = run_silently(
-            'python manage.py behave --help')
+        exit_status, output = run_silently('python manage.py behave --help')
         assert exit_status == 0
         assert (LF + '  --use-existing-database' + LF) in output
 
-    def test_command_should_exit_zero_if_passing(self):
-        exit_status, output = run_silently(
-            'python manage.py behave --tags ~@failing')
-        assert exit_status == 0
+    @patch('behave_django.management.commands.behave.behave_main', return_value=0)  # noqa
+    @patch('sys.exit')
+    def test_command_should_exit_zero_if_passing(self,
+                                                 mock_sys_exit,
+                                                 mock_behave_main):
+        # If the exit status returned by behave_main is 0, make sure sys.exit
+        # does not get called
+        run_management_command('behave', dry_run=True)
```

Comment:
We're also mocking behave, so behave won't actually run in this case.

In my head, the thing we need to test is the behavior of our management command. So in reality, it doesn't matter what behave_main returns at all. What's important is that whatever it may be (exit code 0 or non-zero), we're handling it properly.

The reason why I chose to go with `--dry-run` is so that the test database won't be created. Again, whether the test database is created or not isn't important. That's not part of the scope of the test. :smile: